tools/webinstall/install.sh

```diff
@@ -1188,6 +1207,104 @@ install_kraftkit() {
     fi
 }
 
+# install_dependencies_gnu installs all kraftkit dependencies needed for
+# building and running unikernels on gnu distributions.
+# Returns:
+# Code: 0 on success, 1 on error
+install_dependencies_gnu() {
+    _idd_list=$(printf "%s%s"                       \
+        "build-essential libncurses-dev flex git"   \
+        "wget socat bison unzip qemu-system"        \
+    )
+
+    need_cmd "$AWK"
+    need_cmd "$GREP"
+
+    if check_os_release "rhel" || check_os_release "fedora"; then
+        _idd_list=$(printf "%s%s"                               \
+            "ncurses-devel flex git wget socat bison unzip "    \
+            "qemu-system-x86 qemu-system-arm"                   \
+        )
+        _idd_grp_lst="'C Development Tools and Libraries' 'Development Tools'"
+        _idd_no_weak_deps="--setopt=install_weak_deps=False"
+
+        need_cmd "$YUM"
+
+        do_cmd "$YUM makecache"
+
+        say_debug "Installing dependencies: $_idd_list $_idd_grp_lst"
+        do_cmd "$YUM group install $_idd_no_weak_deps -y $_idd_grp_lst"
+        do_cmd "$YUM install $_idd_no_weak_deps -y $_idd_list"
+    elif check_os_release "debian"; then
+        need_cmd "$APT"
+
```

Comment:
Can you move the debian-specific `_idd_list` list here, similar to how it's defined for other distributions?

Reply:
the error in the default case suggests the packages for manual installation, that's why I kept it up, to avoid code duplication

Reply:
That makes sense.